extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -166,17 +182,8 @@ export function createLabel(view: azdata.ModelView, info: { text: string, descri
 	return text;
 }
 
-export function createNumberInput(view: azdata.ModelView, info: { defaultValue?: string, ariaLabel?: string, min?: number, max?: number, required?: boolean, width?: string, placeHolder?: string }): azdata.InputBoxComponent {
-	return view.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({
-		value: info.defaultValue,
-		ariaLabel: info.ariaLabel,
-		inputType: 'number',
-		min: info.min,
-		max: info.max,
-		required: info.required,
-		width: info.width,
-		placeHolder: info.placeHolder
-	}).component();
+export function createNumberInput(view: azdata.ModelView, info: { defaultValue?: string, ariaLabel: string, min?: number, max?: number, required?: boolean, width?: string, placeHolder?: string, enabled?: boolean, container?: azdata.window.Dialog | azdata.window.Wizard, validations?: Validation[] }): azdata.InputBoxComponent {
```

Comment:
Consider making a type for the info param here - that's a lot of stuff embedded in the signature

Reply:
Ping on this - it'd help readability to have this be a type

Reply:
Done